src/pages/docs/octopus-rest-api/openid-connect/index.md

```diff
@@ -0,0 +1,111 @@
+:::
+
+Any issuer that can generate signed OIDC tokens which can be validated anonymously is supported, however first-class support for GitHub Actions is provided with the [`OctopusDeploy/login`](https://github.com/OctopusDeploy/login) action.
+
+## Getting started with GitHub Actions
+
+To get started using OIDC with GitHub Actions use the below instructions. For more information see [Using OpenID Connect with Octopus and GitHub Actions](./github-actions.md).
+
+### Create an OIDC identity for a service account
+
+The first step is to create an OIDC identity for your GitHub repository to allow workflow runs to access the Octopus API.
+
+1. Go to Configuration -> Users and either create a new service account or locate an existing one.
+2. Open the OpenID Connect section.
+3. Click the New OIDC Identity button.
+4. Select GitHub Actions as the issuer type.
+5. Enter the details of your repository and how you want to filter the workflow runs that can authenticate using OIDC.
+6. Click Save.
+
+:::figure
+![OIDC Identity for GitHub Actions](/docs/octopus-rest-api/images/oidc-identity-github-actions.png "width=500")
+:::
+
+### Add the `OctopusDeploy/login` action to your workflow
+
+After the OIDC identity for GitHub Actions has been created a snippet of the `OctopusDeploy/login` step will be provided which you can use in your workflow to configure the workflow run job to use OIDC authentication.
+
+:::figure
+!['OctopusDeploy/login' snippet](/docs/octopus-rest-api/images/oidc-github-actions-details.png "width=500")
+:::
+
+1. Click the Copy to clipboard to copy the `OctopusDeploy/login` step.
+2. Paste the `OctopusDeploy/login` step into your workflow job.
+3. Add `id-token: write` to the `permissions` on the workflow job. This is required to allow the `OctopusDeploy/login` action to request an OIDC token from GitHub to use.
+   1. Depending on the other steps you have in your workflow, you may need to add additional permissions such as `contents: read`.
+4. Add any additional Octopus provided GitHub Actions that you require e.g. [`OctopusDeploy/create-release-action`](https://github.com/OctopusDeploy/create-release-action). These actions will automatically work with OIDC. Any script steps that use the `octopus` cli will also automatically work with OIDC.
+
+When the workflow runs the `OctopusDeploy/login` action will authenticate with Octopus using OIDC and configure the remainder of the workflow job to work without needing to provide the `server` or `api_key` values.
+
+:::figure
+![Using 'OctopusDeploy/login' in a workflow](/docs/octopus-rest-api/images/oidc-github-actions-example.png "width=500")
+:::
+
+## Getting started with other issuers
+
+To get started using OIDC with other issuers use the below instructions. For more information see [Using OpenID Connect with Other Issuers](./other-issuers.md).
```

Comment:
Why do we split the details for this on this page, then on the other issuers page?
Are we going to repeat the steps here on that page?

Reply:
The idea was that this was supposed to be a minimal "getting started" kind of guide, with further explanation on the specific "other issuers" page. You are right we would most likely repeat the steps in some fashion, but maybe as separate sections with more detail rather than just dot points. I do wonder whether it makes sense to have a getting started for other issuers given how manual it is likely to be for users and different per issuer. It might make more sense for GitHub Actions given its first class.